api/src/main/java/org/apache/cloudstack/api/response/VirtualMachineResponse.java

```diff
@@ -0,0 +1,116 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package org.apache.cloudstack.api.response;
+
+import org.apache.cloudstack.api.BaseResponse;
+import org.apache.cloudstack.api.EntityReference;
+
+import com.cloud.serializer.Param;
+import com.cloud.vm.VirtualMachine;
+import com.google.gson.annotations.SerializedName;
+
+@EntityReference(value = VirtualMachine.class)
+public class VirtualMachineResponse extends BaseResponse {
+    @SerializedName("id")
+    @Param(description = "the ID of the VM")
+    private String id;
+
+    @SerializedName("type")
+    @Param(description = "the type of VM")
+    private String type;
+
+    @SerializedName("name")
+    @Param(description = "the name of the VM")
+    private String name;
+
+    @SerializedName("clusterid")
+    @Param(description = "the Pod ID for the system VM")
+    private String clusterId;
+
+    @SerializedName("clustername")
+    @Param(description = "the Pod name for the system VM", since = "4.13.2")
+    private String clusterName;
+
+    @SerializedName("hostid")
+    @Param(description = "the host ID for the system VM")
+    private String hostId;
+
+    @SerializedName("hostname")
+    @Param(description = "the hostname for the system VM")
+    private String hostName;
+
```

Comment:
would it be good to add more info the response ? but it could lead to some duplications as the UserVmResponse and SystemVmResponse have lots of information.

let's keep it simple like this. If user wants to know more information, use other APIs (listVirtualMachines or listSystemVMs)

Reply:
Yes, I agree to keep it simple